src/App.js

```diff
@@ -20,7 +20,7 @@ function App() {
   return (
     <div className="app-container">
       <Switch>
-        <Route path="/login-redirect-url" component={LoginRedirect} />
+        <Route path="/login-redirect-url/:query_params" component={LoginRedirect} />
```

Comment:
Shouldn't it be `?:query_params` @Aaroneld?

Reply:
You are absolutely correct